dwave/cloud/config/models.py

```diff
@@ -101,6 +111,19 @@ class PollingSchedule(BaseModel, GetterMixin):
     backoff_base: Optional[float] = 1.3
 
 
+class LongPollingSchedule(BaseModel):
+    """Problem status long polling params."""
+
+    #: Long polling strategy
+    strategy: Literal[PollingStrategy.LONG_POLLING] = PollingStrategy.LONG_POLLING
+
+    #: Maximum duration long polling connection is kept open, in seconds.
+    wait_time: Optional[int] = 30
```

Comment:
Feels slightly strange to have this as an `int` and `pause` as a `float`.

Reply:
I agree, but SAPI requires `int` for `timeout` (not sure why). And `pause` is a local sleep (we'll probably remove it eventually, as I don't see many benefits/use cases).

Reply:
Might be worth a comment? :shrug: